cli/src/pcluster/templates/awsbatch_builder.py

```diff
@@ -193,6 +194,10 @@ def _add_job_queue(self):
                     order=1,
                 )
             ],
+            tags={
+                **get_cluster_tags(self.stack_name, raw_dict=True),
+                **get_custom_tags(self.config, raw_dict=True),
+            },
```

Comment:
if possible I'd avoid to explicitly pass tags to all resources but rather rely on CFN propagating such tags to all stack resources.

Reply:
We can improve it in a next patch

Reply:
Removed propagation to AWS Batch resources for now.